applications/luci-app-dawn/luasrc/model/cbi/dawn/dawn_network.lua

```diff
@@ -68,6 +69,7 @@ function s.render(self, sid)
 											if (type(clientvals) == "table") then
 									%>
 										<tr class="tr">
+											<td class="td"><% if (hosts[clientmac] and hosts[clientmac].name) then print(hosts[clientmac].name) end %></td>
```

Comment:
This looks prone to XSS if someone manages to inject an odd hostname into the network. Please use the following expression:

```html
<td class="td"><%= hosts[clientmac] and hosts[clientmac].name and pcdata(hosts[clientmac].name) or "-" %></td>
```

Reply:
Thank you for the very good feedback. It's my first step with lua, now I learned the difference between print and <= ...
I have changed the line.